AEPMessaging/Tests/UnitTests/MessagingRulesEngineTests.swift

```diff
@@ -65,7 +65,21 @@ class MessagingRulesEngineTests: XCTestCase {
         let propositions = try decoder.decode([PropositionPayload].self, from: propString.data(using: .utf8)!)
         
         // test
-        messagingRulesEngine.loadPropositions(propositions)
+        messagingRulesEngine.loadPropositions(propositions, clearExisting: false)
```

Comment:
In the `loadPropositions` tests here, we should also check updates to

- inMemoryPropositions
- propositionInfo
- whether changes are persisted with a call to addPropositionsToCache (if applicable)